R/colors.R

```diff
@@ -10,7 +10,7 @@
 #' @param domain The possible values that can be mapped.
 #'
 #'   For `colorNumeric()` and `colorBin()`, this can be a simple numeric
-#'   range (e.g. `c(0, 100)`); `colorQuantile()` needs representative
+#'   range (e.g., `c(0, 100)`); `colorQuantile()` needs representative
```

Comment:
Ha. Ok. I guess I'm British now! 😆 https://grammarhow.com/eg-comma/

Reply:
Ha - I had no idea that was an American/British thing! Feel free to revert.